docs/content.zh/docs/dev/table/sql/delete.md

```diff
@@ -0,0 +1,201 @@
+tEnv.executeSql("SELECT * FROM Orders").print();
+// Empty set
+```
+{{< /tab >}}
+{{< tab "Python" >}}
+```python
+table_env = StreamTableEnvironment.create(...)
+
+# 注册一个 "Orders" 表
+table_env.executeSql("CREATE TABLE Orders (`user` STRING, product STRING, amount INT)");
+# 插入原始数据
+table_env.executeSql("INSERT INTO Orders VALUES ('Lili', 'Apple', 1), ('Jessica', 'Banana', 2), ('Mr.White', 'Chicken', 3)").wait();
+table_env.executeSql("SELECT * FROM Orders").print();
+# +--------------------------------+--------------------------------+-------------+
+# |                           user |                        product |      amount |
+# +--------------------------------+--------------------------------+-------------+
+# |                           Lili |                          Apple |           1 |
+# |                        Jessica |                         Banana |           2 |
+# |                       Mr.White |                        Chicken |           3 |
+# +--------------------------------+--------------------------------+-------------+
+# 3 rows in set
+# 根据where条件删除
+table_env.executeSql("DELETE FROM Orders WHERE `user` = 'Lili'").wait();
+table_env.executeSql("SELECT * FROM Orders").print();
+# +--------------------------------+--------------------------------+-------------+
+# |                           user |                        product |      amount |
+# +--------------------------------+--------------------------------+-------------+
+# |                        Jessica |                         Banana |           2 |
+# |                       Mr.White |                        Chicken |           3 |
+# +--------------------------------+--------------------------------+-------------+
+# 2 rows in set
+# 全表删除
+table_env.executeSql("DELETE FROM Orders").wait();
+table_env.executeSql("SELECT * FROM Orders").print();
+# Empty set
+```
+{{< /tab >}}
+{{< tab "SQL CLI" >}}
+```sql
+Flink SQL> CREATE TABLE Orders (`user` STRING, product STRING, amount INT) with (...);
+[INFO] Execute statement succeed.
+
+Flink SQL> INSERT INTO Orders VALUES ('Lili', 'Apple', 1), ('Jessica', 'Banana', 1), ('Mr.White', 'Chicken', 3);
+[INFO] Submitting SQL update statement to the cluster...
+[INFO] SQL update statement has been successfully submitted to the cluster:
+Job ID: bd2c46a7b2769d5c559abd73ecde82e9
+
+Flink SQL> SELECT * FROM Orders;
+    user                         product      amount
+    Lili                           Apple           1
+ Jessica                          Banana           2
+Mr.White                         Chicken           3
+
+Flink SQL> DELETE FROM Orders WHERE `user` = 'Lili';
+
+    user                         product      amount
+ Jessica                          Banana           2
+Mr.White                         Chicken           3
+
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+{{< top >}}
+
+## 语法
+
+```sql
+DELETE FROM [catalog_name.][db_name.]table_name [ WHERE boolean_expression ]
```

Comment:
Have a look about the syantax desc in other engines, like [PG](https://www.postgresql.org/docs/current/sql-delete.html), [SqlSever](https://learn.microsoft.com/en-us/sql/t-sql/statements/delete-transact-sql?view=sql-server-ver16).
How about using 
```
DELETE FROM [catalog_name.][db_name.]table_name [ WHERE condition ]
```
?